src/pudl/metadata/fields.py

```diff
@@ -1034,12 +1034,9 @@
         "type": "string",
         "description": "Coal mine name."
     },
-    "mine_type": {
+    "mine_type_code": {
         "type": "string",
         "description": "Type of coal mine.",
-        "constraints": {
-            "enum": list(COALMINE_TYPES_EIA.values()),
-        }
```

Comment:
Just for my understanding, this field no longer needs an enum constraint because `COALMINE_TYPES_EIA` now lives in `codes.py` and the data is corrected using an encoder? Does it make sense to leave the enum constraint here to double-check?

Reply:
Oh whoops. That should definitely have been removed. I will get it.

Reply:
Not quite -- the coding tables are related to the main data tables by a foreign key constraint. So if there's a bad code, the FK constraint will be violated, and having both an ENUM and FK constraint seemed duplicative.

Reply:
Got it thanks 👍